compiler/src/dotty/tools/dotc/reporting/ErrorMessageID.scala

```diff
@@ -223,6 +223,7 @@ enum ErrorMessageID(val isActive: Boolean = true) extends java.lang.Enum[ErrorMe
   case IllegalUnrollPlacementID // errorNumber: 207
   case ExtensionHasDefaultID // errorNumber: 208
   case FormatInterpolationErrorID // errorNumber: 209
+  case ValueClassCannotExtendAliasOfAnyValID // errorNumber: 210
```

Comment:
awesome errors is an endless source of conflict.
https://github.com/scala/scala3/pull/23002/files